finalPrep/modZeroHero.js

```diff
@@ -55,30 +92,86 @@ var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.
 //   - citiesDestroyed (array)
 //   - luckyNumbers (array)
 //   - address (object with following key/values: number , street , state, zip)
-
+var scaryMonster = {
+  name: "Godzilla",
+  smell: "Stinky",
+  weight: 500,
+  citiesDestroyed: ["Cancun", "Miami"],
+  luckyNumbers: [7, 21, 3],
+  address: {
+    number: 12,
+    street: "16th Street",
+    state: "Kansas",
+    zip: 80203
+  }
+};
 
 // Create a new class called SuperHero
 // - Your class should have the following DYNAMIC values
-//   - name 
+//   - name
 //   - superpower
-//   - age 
+//   - age
 // - Your class should have the following STATIC values
 //   - archNemesis, assigned to "The Syntax Error"
 //   - powerLevel = 100
-//   - energyLevel = 50 
+//   - energyLevel = 50
+class SuperHero {
+  constructor(name, superpower, age){
+    this.archEnemies = "The Syntax Error";
+    this.powerLevel = 100;
+    this.energyLevel = 50;
+    this.name = name;
+    this.superpower = superpower ;
+    this.age = age;
+  }
+}
+var batman = new SuperHero("Bruno", "Martial Arts", 45);
 
 // - Create the following class methods
 //   - sayName, should print the hero's name to the console
 //   - maximizeEnergy, should update the energyLevel to 1000
 //   - gainPower, should take an argument of a number and INCREASE the powerLevel by that number
+class SuperHero {
+  constructor(name, superpower, age){
+    this.archEnemies = "The Syntax Error";
+    this.powerLevel = 100;
+    this.energyLevel = 50;
+    this.name = name;
+    this.superpower = superpower ;
+    this.age = age;
+  }
+    sayName() {
+    console.log(this.name);
+  } maximizeEnergy() {
+    this.energyLevel = 1000;
+  } gainPower() {
```

Comment:
According to the instructions, `gainPower` should take an argument of a number, so I expected to see something like this:
```
gainPower(power) {
   this.powerLevel = this.powerLevel + power;
}
```